Tools/WinMLRunner/Main.cpp

```diff
@@ -296,12 +296,6 @@ HRESULT EvaluateModels(
             {
                 for (auto inputDataType : inputDataTypes)
                 {
-                    if (args.PerfCapture())
```

Comment:
The perf data was getting cleared before we could even write the load time, so I placed it at the end of the loop instead.